pxr/usdImaging/usdImaging/delegate.cpp

```diff
@@ -282,11 +282,17 @@ UsdImagingDelegate::_GetHdPrimInfo(const SdfPath &cachePath)
 }
 
 Usd_PrimFlagsConjunction 
-UsdImagingDelegate::_GetDisplayPredicate() const
+UsdImagingDelegate::_GetDisplayPredicate(bool forPrototypes) const
 {
-    return _displayUnloadedPrimsWithBounds ?
-            UsdPrimIsActive && UsdPrimIsDefined && !UsdPrimIsAbstract :
-            UsdPrimDefaultPredicate;
+    Usd_PrimFlagsConjunction retval = UsdPrimIsActive && !UsdPrimIsAbstract;
```

Comment:
This is the same as
```
          UsdPrimIsActive && !UsdPrimIsAbstract &&
          (forPrototypes || UsdPrimIsDefined) &&
          (_displayUnloadedPrimsWithBounds || UsdPrimIsLoaded);
```
Might make it easier to figure out what the patch does.

Reply:
Thanks Bill-- to me, it's more clear as is, but I'm happy to change it per your suggestion if there's a consensus to do so. Thanks for looking!